src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -155,19 +155,19 @@ def getArchConf(config, arch) {
     k8sArchConfTable['x86_64']  = [
         nodeSelector: 'kubernetes.io/arch=amd64',
         jnlpImage: 'jenkins/inbound-agent:latest',
-        dockerImage: 'docker:19.03'
+        dockerImage: 'harbor.mellanox.com/swx-storage/ci-demo/podman:v5.0.2'
```

Comment:
why from harbor? can we use it from public registry to simplify?

Reply:
The public registry is rate-limited, leading to CI failures.

Reply:
Sure, but I wouldn't want to place it into someone's personal account in Docker Hub.
Maybe we already have something in the public registry for such things?

Reply:
registry path is set by ```registry_path``` var and it will generate harbor or global registry url
https://github.com/Mellanox/ci-demo/blob/master/src/com/mellanox/cicd/Matrix.groovy#L1150

no need to embed mellanox specific registry, it is controlled by config var registry_path of job_matrix.yaml file, if set - use harbor, if not - use global from docker